PhysicsTools/PatAlgos/plugins/PATLostTracks.cc

```diff
@@ -190,10 +196,28 @@ void pat::PATLostTracks::produce(edm::StreamID, edm::Event& iEvent, const edm::E
     }
   }
   for (const auto& v0 : *lambdas) {
+    double protonCharge = 0;
     for (size_t dIdx = 0; dIdx < v0.numberOfDaughters(); dIdx++) {
       size_t key = (dynamic_cast<const reco::RecoChargedCandidate*>(v0.daughter(dIdx)))->track().key();
       if (trkStatus[key] == TrkStatus::NOTUSED)
         trkStatus[key] = TrkStatus::VTX;
+      protonCharge += v0.daughter(dIdx)->charge() * v0.daughter(dIdx)->momentum().mag2();
+    }
+    if (xiSelection_) {
+      // selecting potential Xi- -> Lambda pi candidates
+      TLorentzVector p4Lambda;
```

Comment:
please switch to a `math::` Lorentz vector. Given the cost of computing eta and phi for tracks, I think that `math::XYZTLorentzVector` is more appropriate.